### PR TITLE
i18n(zh-tw): update `astro-pages.mdx` translation

### PR DESCRIPTION
**Pull request by mingjunlu**
#### Description

- Updates `src/content/docs/zh-tw/basics/astro-pages.mdx`

#### Related issues & labels

- ~~Closes #~~
- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/zh-tw/basics/astro-pages.mdx [hunk 1]
```diff
@@ -1,35 +1,47 @@
...
--  `.mdx` （在安裝 [MDX 整合](/zh-tw/guides/integrations-guide/mdx/#installation)的情況下）
+- `.mdx`（在安裝 [MDX 整合](/zh-tw/guides/integrations-guide/mdx/#installation)的情況下）
 - [`.html`](#html-頁面)
-- `.js`/`.ts` （作為 [endpoints](/zh-tw/guides/endpoints/)）
+- `.js`/`.ts`（作為[端點](/zh-tw/guides/endpoints/)）
 
 ## 基於檔案的路由
 
-Astro 使用了一個路由策略叫 **基於檔案的路由（file-based routing）** 。每個在你 `src/pages/` 目錄下的檔案都會變成你網站的 endpoint，而其會根據檔案自己的路徑而定。
+Astro 使用**基於檔案的路由（file-based routing）**策略。每個 `src/pages/` 目錄下的檔案，都會根據檔案的路徑，變成網站的端點。
```
Comment (liruifengv) [inline src/content/docs/zh-tw/basics/astro-pages.mdx:23]: 这里加粗的md格式渲染错误

**Review by liruifengv — CHANGES_REQUESTED** [review]
Thank you update @mingjunlu There have a small issue.

File: src/content/docs/zh-tw/basics/astro-pages.mdx (hunk 1 above)
Comment (mingjunlu) [reply src/content/docs/zh-tw/basics/astro-pages.mdx]: 感謝，已調整：5aa8e9bacaaf76b3471254ea39f6c18c4c0828e9

**Review by liruifengv — APPROVED** [review]
LGTM

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks 🙌

**Pull request merged**